test/test_regression_cli.py

```diff
@@ -46,7 +46,6 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir("./test/")
```

Comment:
for some reason, it was causing some errors, I will go ahead and test it once again and let you know.